src/kudu/consensus/consensus_meta.h

```diff
@@ -193,6 +188,8 @@ class ConsensusMetadata : public RefCountedThreadSafe<ConsensusMetadata> {
   FRIEND_TEST(ConsensusMetadataTest, TestToConsensusStatePB);
   FRIEND_TEST(ConsensusMetadataTest, TestMergeCommittedConsensusStatePB);
 
+  static const int32_t VOTE_HISTORY_MAX_SIZE = 10000;
```

Comment:
oops :-) can we reduce it to something more reasonable like 20

Reply:
Reduced to 100 :) I was thinking about this as 100 bytes per entry and roughly <10MiB for 100k entries.